installers/osie/main.go

```diff
@@ -122,6 +130,12 @@ func (i installer) rescue(ctx context.Context, j job.Job, s *ipxe.Script) {
 }
 
 func (i installer) discover(ctx context.Context, j job.Job, s *ipxe.Script) {
+	if i.extraIPXEVars != nil {
+		for index := 0; index < len(i.extraIPXEVars); index++ {
```

Comment:
any reason this loop is different from the others?

Reply:
This was my original approach before I learned about using range for this, I had just missed updating it. This should now be fixed.